src/test/java/io/fabric8/maven/docker/config/handler/compose/ComposeUtilsTest.java

```diff
@@ -0,0 +1,50 @@
+package io.fabric8.maven.docker.config.handler.compose;
+
+import org.apache.maven.project.MavenProject;
+import org.junit.Test;
+
+import java.io.File;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
```

Comment:
would it be able to use jmockit as Mock framework (as we already use this and I'm not so inclined to use multiple mock frameworks in d-m-p)

You could take e.g. https://github.com/rhuss/docker-maven-plugin/blob/674e80fdf56d31fdd263366d53e7a5cf0dc064b3/src/test/java/io/fabric8/maven/docker/access/log/LogRequestorTest.java#L220 as a sample how expectations are setup in jmockit.